manifests/topic.pp

```diff
@@ -32,17 +35,29 @@
 #   See the Kafka documentation for full details on the topic configs.
 #
 define kafka::topic (
-  String[1] $ensure                           = '',
-  String[1] $zookeeper                        = '',
+  String[1] $ensure                           = undef,
+  Optional[String[1]] $zookeeper              = undef,
+  Optional[String[1]] $bootstrap_server       = undef,
```

Comment:
Allowing Strings here could a potential security risk due to the `exec` resource if I'm not mistaken.
An issue for another PR, but this could be limited/sanitized with a custom data type

Reply:
As you said, I think this is out of scope for this PR :slightly_smiling_face: